Documentation/manualtest-guide.md

```diff
@@ -0,0 +1,12 @@
+Manual Tests Instruction
+========================
+Test Name: WindowsAuthentication_RoundTrips_Echo
+Set Up Requirement
+1. Must run on a domain machine.
+
+Steps
+1. Enable the test
+In src\System.Private.ServiceModel\tests\Scenarios\Security\TransportSecurity\Https\ClientCredentialTypeTests.cs,
+Remove  [ActiveIssue(53)] above test method WindowsAuthentication_RoundTrips_Echo()
+2. Go to the root of WCF source, run all scenario tests.
+	build /p:WithCategories=OuterLoop
```

Comment:
> We don't have a way to specify categories of tests to run. Waiting on the build-tools to accommodate this.

Not "categories" of test - just particular "test" e.g., spin up the bridge, and build *this specific csproj* using msbuild to run the test.